google/cloud/bigtable/data/_async/_read_rows.py

```diff
@@ -59,158 +59,253 @@ class _ReadRowsOperationAsync(AsyncIterable[Row]):
 
```

Comment:
Do you need to add slots? I think _last_yielded_row_key will accessed fairly often

Reply:
Good point, fixed. I didn't see a noticeable a perf bump in my run after adding slots, but still seems like a good change